prow/cluster/jobs/istio/tools/istio.tools.yaml

```diff
@@ -0,0 +1,74 @@
+presubmits:
+
+  istio/tools:
+
+    - name: tools-lint
+      path_alias: istio.io/tools
+      decorate: true
+      always_run: true
+      optional: true
+      spec:
+        containers:
+          - image: gcr.io/istio-testing/circleci:2019-06-13
+            command:
+              - make
+              - lint
+            env:
+              - name: GO111MODULE
+                value: "on"
+        nodeSelector:
+          testing: test-pool
+
+    - name: tools-build
+      path_alias: istio.io/tools
+      decorate: true
+      always_run: true
+      optional: true
+      spec:
+        containers:
+          - image: gcr.io/istio-testing/circleci:2019-06-13
+            command:
+              - go
+              - build
+              - ./...
+            env:
+              - name: GO111MODULE
+                value: "on"
+        nodeSelector:
+          testing: build-pool
```

Comment:
I think build-pool is only needed if you intend to push a gcr.io image.  this point is a little vague to me, but I don't think it particularly matters, other than capacity of the two pools.